backend/src/core/models/game.spec.js

```diff
@@ -587,4 +636,47 @@ describe("Game", function () {
       });
     });
   });
+
+  describe(".updateReleaseDateViaSteamApi", function () {
+    describe("If the game already has an existing release date", function () {
+      beforeAll(function () {
+        this.game = getXGamesWithoutDetails(1)[0];
+        this.existingDate = new Date("23 July 2023");
+
+        this.game.releaseDate = this.existingDate;
+
+        this.game.updateReleaseDateViaSteamApi(eldenRingSteamApiData);
+      });
+
+      it("the game's release date stays unchanged", function () {
+        expect(this.game.releaseDate).toBe(this.existingDate);
+      });
+    });
+
+    describe("When we try to use a page that has no existing release date", function () {
```

Comment:
There are no more pages, you are using the API now. Update it to talk about a "raw steam app data set". You can also update it in the tests above and below if there is something to update.